grammars/hrm.cson

```diff
@@ -3,20 +3,20 @@
 'fileTypes': [
   'hrm'
 ]
-'firstLineMatch': '^-- HUMAN RESOURCE MACHINE PROGRAM --$'
+'firstLineMatch': '^ *-- HUMAN RESOURCE MACHINE PROGRAM --'
 'patterns': [
   {
-    'match': '--.*$'
+    'match': ' *--.*'
```

Comment:
I don't _think_ this change is correct…  Could you send me a file that imports correctly but has the wrong highlighting without this change, but the right highlighting with it?

(My understanding of comments is that they can start anywhere and go to the end of the line…  So `label:--Comment` is valid, but wouldn't be highlit by the changed line.)